.azure-pipelines/generation-templates/generate-helpdocs-template.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+parameters:
+  - name: DOCSBRANCH
+    type: string
+    default: "DocsGeneration"
+  - name: ModulesToGenerate
```

Comment:
Parameter `ModulesToGenerate` is declared but never used anywhere in this template. Shouldn't we be passing it as a parameter to `. $(System.DefaultWorkingDirectory)/tools/GenerateHelp.ps1`?

Reply:
See suggestion -> https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/726#discussion_r662648181.